ocaml/xapi/updateinfo.mli

```diff
@@ -162,9 +162,9 @@ module UpdateInfo : sig
 
   val guidance_to_string : Guidance.t option -> string
 
-  val of_xml : Xml.xml -> (string * t) list
+  val of_xml : Xml.xml -> string option * (string * t) list
```

Comment:
(I don't have a strong opinion on this.)
The first `string option` may work for the only data api version. But I think it would be better to use a record to wrap it, although there is only one data in it at present.

Reply:
What I mean is something like `val of_xml : Xml.xml -> UpdatesAttr.t * (string * UpdateInfo.t) list` in which `UpdateInfoAttr.t ` is for type of the attributes of `updates` node.